src/controller/CHIPDeviceController.h

```diff
@@ -192,6 +200,7 @@ class DLL_EXPORT DeviceController : public SecureSessionMgrDelegate, public Pers
     Messaging::ExchangeManager * mExchangeManager;
     PersistentStorageDelegate * mStorageDelegate;
     Inet::InetLayer * mInetLayer;
+    Ble::BleLayer * mBleLayer = nullptr;
```

Comment:
BleLayer doese not exist if CONFIG_NETWORK_LAYER_BLE is false. We have to guard these.